package.json

```diff
@@ -13,7 +13,8 @@
   "scripts": {
     "build": "preconstruct build",
     "test": "karma start",
-    "semantic-release": "semantic-release"
+    "semantic-release": "semantic-release",
+    "prepublishOnly": "npm run build"
```

Comment:
is this to avoid having the build script being executed twice?

on another issue, why do we use `npx` to run the semantic-release instead of `npm`?

Reply:
No, it's to be able to run `build` after the version has been decided. Removing it from `release.yml` is to avoid executing it twice.

`npx` is purely because it was copied from the [official example](https://github.com/semantic-release/semantic-release/blob/master/docs/recipes/ci-configurations/github-actions.md#githubworkflowsreleaseyml-configuration-for-node-projects). `npx` will run the locally installed package, so it doesn't fetch anything remotely, but I will change to `npm` anyway, as that will be more explicit.